W-16539732 RPA Manager release notes 2024-08/2. (#412)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[august-21-2024]]
== August 21, 2024

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * A new permission has been added to control the transition from test in production phase. (W-16247708)

Reviewer: ```suggestion
* The new *Process publish restriction for Productionphase* permission prevents a user from moving a process from Testing to Production. 
```

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | The Process Management's *Categories* page now shows only one banner. | W-14436611

Reviewer: ```suggestion
| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
```

> | Now the *Process Manager* is shown correctly in the list of *Processes for Finance Analysis*. | W-14851495

Reviewer: ```suggestion
| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
```

> | During the creation of a new *Evaluation* now an input character counter is shown. | W-15301328

Reviewer: ```suggestion
| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
```

> | The assigned applications are no longer deleted during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363

Reviewer: ```suggestion
| RPA Manager no longer deletes assigned applications during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
```

> | RPA Bots can now be updated in parallel. | W-16382151

Reviewer: ```suggestion
| You can now update multiple RPA Bots in parallel. | W-16382151
```

> | During the creation of an *Evaluation* a new *Category* can now be created. | W-16393605

Reviewer: ```suggestion
| You can no longer create a new evaluation category with an empty name. | W-16393605
```

Author: That's correct.

> | During the creation of an *Evaluation* a new *Category* can now be created. | W-16393605

Reviewer: ```suggestion
| You can no longer create a *Category* with an empty name. | W-16393605
```

> | It is no longer possible for multiple RPA Bots to receive the same task for executing the same run configuration. | W-16436289

Reviewer: ```suggestion
| RPA Bots can no longer receive the same task for executing the same run configuration. | W-16436289
```

> | An *Alert Rule* now keeps its activation status even after a modification. | W-16528085

Reviewer: ```suggestion
| Editing a deactivated *Alert Rule* no longer reactivates it. | W-16528085
```
